docs/content/gameobject/gomb-creating-workers-with-workerconnector.md

```diff
@@ -1,39 +1,42 @@
 [//]: # (Doc of docs reference 15.1)
 
 # (GameObject-MonoBehaviour) Creating workers with WorkerConnector
-_This document relates to the [GameObject-MonoBehaviour workflow](./intro-workflows-spos-entities.md)._
+_This document relates to the [GameObject-MonoBehaviour workflow]({{urlRoot}}/content/intro-workflows-spos-entities)._
 
 See first the documentation on:
-* [Workers in the GDK](../workers/workers-in-the-gdk.md)  
-* [Worker API](../workers/api-worker.md)
+
+* [Workers in the GDK]({{urlRoot}}/content/workers/workers-in-the-gdk)  
+* [Worker API]({{urlRoot}}/content/workers/api-worker)
 
 To demonstrate use of the `Worker` class, the GDK contains an example implementation
 of how to create your workers. We have implemented this in
 the `WorkerConnector` class which you can extend further by creating classes which inherit from it.
 The `WorkerConnector` is a MonoBehaviour script. You can use it to create multiple workers
 in one Scene by adding it to multiple GameObjects, each GameObject creating a different worker.
 
-When you have multiple workers represented as GameObjects in a scene, you are likely to have multiple ECS entities [checked out](../glossary.md#authority) to that worker. To make sure you don’t have all the ECS entities which are checked out to a worker/GameObject in the same x,y,z location, we use an offset for each ECS entity against the origin of the worker/GameObject.  We call the offset of the worker/GameObject origin, the “translation”.
+When you have multiple workers represented as GameObjects in a scene, you are likely to have multiple ECS entities [checked out]({{urlRoot}}/content/glossary.md#authority) to that worker. To make sure you don’t have all the ECS entities which are checked out to a worker/GameObject in the same (x, y, z) location, we use an offset for each ECS entity against the origin of the worker/GameObject.  We call the offset of the worker/GameObject origin, the “translation”.
 
 
 ## How to use worker prefabs
 
-In the GDK’s [`Playground` project](../../workers/unity/Assets/Playground), we provide
+In the GDK’s [`Playground` project](https://github.com/spatialos/gdk-for-unity/workers/unity/Assets/Playground), we provide
```

Comment:
`https://github.com/spatialos/gdk-for-unity/tree/master/workers/unity/Assets/Playground`

Reply:
Fixed.